hadoop-shims/common/pom.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Could we put the hadoop-shims into client-mr module?

Reply:
We use the directory structure like 
```
client-mr
    |---- core
    |---- hadoop-shims
```

Reply:
It is an issue which we need to discuss. We know many module use org.apache.hadoop:hadoop-common. If some incompatibility problem happen in hadoop-common, and it is not proper that hadoop-shims is the sub module of client-mr.

Reply:
If so, we should put the module to the common like
```
common
    |----- core 
    |---- hadoop-shims
```
Now, we don't encounter the incompatibility issues of hadoop-common, we can put it into mr module. But if you think we must encounter the problem, we can put it into common module.

Reply:
I don't know what's common/core? Is it just the common module? We move the common module to common/core directory?